README.RDAP.md

```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 Related Contact information is Filtered
 ---------------------------------------
 All contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
-and "notify" values are not included. Contact entities do not count towards the daily query limit
+value is not included. Contact entities do not count towards the daily query limit
 https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use-policy.
 This was done to avoid blocking clients for inadvertently querying excessively for personal data.
 
-For entity request "e-mail" will be enabled and the clients requesting those service will need to comply with the daily limit
+For entity requests "e-mail" is enabled and the clients requesting this service need to comply with the daily limit
 according to the Acceptable User Policy (AUP): https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use-policy.
+So this is the way to get the "e-mail" of the contact entities.
 
-For the rest of request either "e-mail" is filtered. Abuse contact is always returned, and
+For the rest of the requests either "e-mail" is filtered. Abuse contact is always returned, and
```

Comment:
"For non-entity requests, e-mail addresses are filtered, except for the abuse contact which is always returned."

Separate sentence for the other attribute types.